legal-api/src/legal_api/reports/report.py

```diff
@@ -1078,7 +1078,8 @@ def _get_legal_type_description(legal_type):
         return corp_type.full_desc if corp_type else None
 
     def _format_correction_data(self, filing):
-        if bool(filing.get('correction', {}).get('commentOnly', False)):
+        if bool(filing.get('correction', {}).get('commentOnly', False)) or \
+           bool(filing.header.get('correctionBenStatement', False)):
```

Comment:
Added this check to skip rest of the code run when running BEN statement corrections